lib/Provider/future_provider.dart

```diff
@@ -163,18 +164,34 @@ class BrightnessNotifier extends AutoDisposeNotifier<Brightness>
     with WidgetsBindingObserver {
   @override
   void didChangePlatformBrightness() {
-    state = WidgetsBinding.instance.platformDispatcher.platformBrightness;
+    changeBrightness(
+        WidgetsBinding.instance.platformDispatcher.platformBrightness ==
+            Brightness.dark);
     super.didChangePlatformBrightness();
   }
 
   @override
   Brightness build() {
+    bool? isDark = Config().pref.getBool("isDark");
     WidgetsBinding.instance.addObserver(this);
-    return WidgetsBinding.instance.platformDispatcher.platformBrightness;
+    if (isDark == null) {
+      Brightness brightness =
+          WidgetsBinding.instance.platformDispatcher.platformBrightness;
+      brightness == Brightness.dark
+          ? Config().pref.setBool("isDark", true)
```

Comment:
Isn't this an async operation?, just having that without await seems risky. I think it better if you can change this to a future provider.